forem/forem
# GitHub liquid tag: add support for img tags with no src attribute

## What type of PR is this? (check all applicable)


## Description

Fixes https://github.com/forem/forem/issues/10469

I concentrated on following the issue at hand, even though that method could probably be simplified

## QA Instructions, Screenshots, Recordings

1. Add `{% github sirixdb/sirix %}` on https://dev.to/new and see it break
1. Add `{% github sirixdb/sirix %}` on this branch and see it work

I added a regression test ;-)

## Added tests?
